<launch>

    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true">
        <param name = "path" value = "$(find path_plan)/path.txt"/>
        <param name = "path_window_size" value = "1.5"/>
        <param name = "planner_frequency" value = "1.0"/>
        <param name="base_global_planner" value="path_planner/GlobalPlanner"/>
        <!-- <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS"/> -->
        <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS"/>

        <rosparam file="$(find path_plan)/param/navigation/costmap/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find path_plan)/param/navigation/costmap/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find path_plan)/param/navigation/costmap/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find path_plan)/param/navigation/costmap/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find path_plan)/param/navigation/planner/base_local_planner_params.yaml" command="load" />
        <rosparam file="$(find path_plan)/param/navigation/planner/global_planner_params.yaml" command="load" />
        <rosparam file="$(find path_plan)/param/navigation/planner/dwa_local_planner_params.yaml" command="load" />
        <rosparam file="$(find path_plan)/param/navigation/planner/teb_local_planner_params.yaml" command="load" />
        <rosparam file="$(find path_plan)/param/navigation/planner/costmap_converter_params.yaml" command="load" />

        
    </node>

</launch>
